gnomad/resources/grch37/gnomad.py

```diff
@@ -156,6 +156,22 @@ def _public_browser_gene_ht_path() -> str:
     return "gs://gnomad-public-requester-pays/resources/grch37/browser/gnomad.genes.GRCh37.GENCODEv19.pext.ht"
 
 
+def _public_mnv_path(distance: str = "1") -> str:
+    """
+    Get public multinucleotide variant (MNV) data.
+
+    :param distance: Distance between two SNVs in MNV. Default is '1'.
+    :return: Path to MNV data.
+    :raises DataException: If the provided distance is invalid (outside 1-10).
```

Comment:
is this something we're adding now or can the "raises" part here be dropped?

Reply:
ah I copied the structure from pext above and am happy to remove (maybe cc @mike-w-wilson for opinions?)